cpp/src/slice2cs/Gen.cpp

```diff
@@ -404,8 +404,9 @@ Slice::CsVisitor::writeDispatch(const InterfaceDefPtr& p)
     _out << sp;
     _out << nl << "public override bool ice_isA(string s, " << getUnqualified("Ice.Current", ns) << " current = null)";
     _out << sb;
-    _out << nl
-         << "return global::System.Array.BinarySearch(_ids, s, IceUtilInternal.StringUtil.OrdinalStringComparer) >= 0;";
+    _out
+        << nl
+        << "return global::System.Array.BinarySearch(_ids, s, Ice.UtilInternal.StringUtil.OrdinalStringComparer) >= 0;";
```

Comment:
Not related to the PR but why don't we use `StringComparer.Ordinal` https://learn.microsoft.com/en-us/dotnet/api/system.stringcomparer.ordinal?view=net-8.0

Reply:
Yes, we should fix that.